lib/plugins/blocks.js

```diff
@@ -275,7 +275,7 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
       bitMap: packet.bitMap,
       heightmaps: packet.heightmaps,
       biomes: packet.biomes,
-      skyLightSent: bot.game.dimension === 'minecraft:overworld',
+      skyLightSent: bot.game.dimension === 'overworld',
```

Comment:
This should be inferred from the dimension codec

Reply:
Yeah, that can be addressed in a different PR, let's just normalize the output values for all versions first